components/modal/modal.css

```diff
@@ -0,0 +1,41 @@
+.modal-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
you can extract this in a utility class so that it can be used elsewhere as well

Reply:
yes, you are right I will change it while refactoring the component library.Thank you